README.adoc

```diff
@@ -32,9 +32,17 @@ It is very easy to analyze a C, C++ and Objective-C project with SonarQube on Az
 * Choose "Use standalone scanner" (`scannerMode: 'CLI'`)
 * Choose "Manually provide configuration"
 * Specify the "Project Key" and the "Sources directory root"
-* In "Additional Properties" in the "Advanced" section, add the property `sonar.cfamily.build-wrapper-output` with, as its value, the output directory to which the Build Wrapper should write its results (e.g. `build_wrapper_output_directory`)
+* In "Additional Properties" in the "Advanced" section, add
++
+--
+** the property `sonar.cfamily.compile-commands` with, as its value, `bw-output/compile_commands.json`, if you are using SonarQube version 10.6 or later 
+** the property `sonar.cfamily.build-wrapper-output`, with value `bw-output`, if you are using SonarQube version 10.5 or earlier, as build-wrapper did not generate a `compile_commands.json` file before SonarQube 10.6
+** 
+--
++
+Noting that `bw-output` is the directory passed to build wrapper as `--out-dir`
 .. Add a task to download the https://docs.sonarqube.org/latest/analyzing-source-code/languages/c-family/#using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
+.. Wrap your compilation with the Build Wrapper, specifying the output directory `out-dir`
```

Comment:
```suggestion
.. Wrap your compilation with the Build Wrapper, specifying the output directory `bw-output`
```

Reply:
> I would suggest something like this instead (we already have a comment above about bw-output). Let me know what you think.

Actually I think this is somewhat confusing, as in one of the case you are passing the compilation database and not the output folder in the additional properties

Reply:
Fair point. There is a sentence at the end that mentions the output directory specifically after both cases, so I am trying to refer to that sentence here.

Let me know if you have other suggestions to avoid confusion. We can also discuss them in a call if you find this easier...